hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java

```diff
@@ -966,6 +966,22 @@ public static ServerName getServerName(final Result r, final int replicaId) {
     }
   }
 
+  /**
+   * Returns a host:port string of server where the region is transitioning on.
+   * It reads "sn" column from {@link Result}, a row in catalog table.
+   * @param r Result to pull from
+   * @return A host:port string or null if necessary fields not found or empty.
+   */
+  @Nullable
+  @InterfaceAudience.Private // for use by RegionReplicaInfo which is used by meta browser
+  public static String getTransitioningOnServerName(final Result r, final int replicaId) {
```

Comment:
A small test case to cover this would be good. Maybe in `TestMaster`.

Reply:
I navigate the source code tree, and found another place has implemented the same logic there. So in recent commit I simply move the code to `MetaTableAccessor.java` so it can be reused. I hope I have not broken any stuff, so wait for the QA to confirm. Since that is exactly the existing method, I guess a test is not required? But I can work on a UT if you still suggest. Thanks,